server/autotest_enqueuer.py

```diff
@@ -32,6 +32,15 @@ def check_args(func, *args, **kwargs):
     except TypeError as e:
         raise type(e)('{}\nWith args: {}\nWith kwargs:{}'.format(e, args, tuple(kwargs))).with_traceback(sys.exc_info()[2])
 
+def check_for_environment_errors(markus_address, tester_type, tester_name, **kw):
```

Comment:
@mishaschwartz I get an error after your kw commit:
check_for_environment_errors(**kw)
TypeError: check_for_environment_errors() missing 2 required positional arguments: 'tester_type' and 'tester_name'

Do they need to be passed from Markus? What about the basic case in the existing setup, where we upload our own scripts without using a pre-installed tester?